src/ansys/speos/core/generic/constants.py

```diff
@@ -75,6 +77,30 @@ class INTENSITY:
         VALUE = 5
 
 
+@dataclass(frozen=True)
+class FluxLuminous:
+    """Constant class for Luminous type Flux."""
+
+    value: float = 683
+
+
+@dataclass(frozen=True)
+class FluxRadiant:
+    """Constant class for Radiant type Flux."""
+
+    value: float = 1
+
+
+@dataclass(frozen=True)
+class SourceRayfileParamters:
+    """Constant class for SourceRayfileParamters."""
+
+    flux_from_ray_file: bool = True
+    flux_luminous: Union[bool, FluxLuminous] = FluxLuminous()
+    flux_radiant: Union[bool, FluxRadiant] = False
+    axis_system: list[float] = field(default_factory=lambda: [0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 1])
+
+
```

Comment:
@StefanThoene @echambla @SMoraisAnsys could you please provide your feedback about the new dataclass structure for default parameters?